components/builder-originsrv/src/server/mod.rs

```diff
@@ -165,3 +164,13 @@ impl Dispatcher for OriginSrv {
 pub fn run(config: Config) -> AppResult<(), SrvError> {
     app_start::<OriginSrv>(config)
 }
+
+pub fn migrate(config: Config) -> SrvResult<()> {
+    // Why does it not matter? Because we're not staying alive and communicating with other
+    // services. We're just running migrations and quitting.
+    let router_pipe = Arc::new(format!(
+        "this-is-not-a-real-router-pipe-but-it-doesnt-matter!"
```

Comment:
This _might_ need to be in a proper format - eg, "inproc:// ..."

Reply:
It worked in my local tests, where by "worked" I mean migrations ran.

Reply:
👍

Reply:
I'll change it anyway, though, because it can't hurt.